core/celo_multi_gaspool.go

```diff
@@ -48,12 +61,12 @@ func NewMultiGasPool(
 
 // PoolFor returns a configured pool for the given fee currency or the default
 // one otherwise
```

Comment:
```suggestion
// PoolFor returns a configured pool for the given fee currency or the default
// one otherwise, the returned boolean is true when a custom fee currency gas pool is returned.
```